src/util/submit.rs

```diff
@@ -8,12 +8,12 @@ use crate::error::AocError;
 
 pub fn get_submit_day(matches: &ArgMatches) -> Option<Result<Task, AocError>>
```

Comment:
You don't need to do anything about if for this PR, but I just noticed that I think the name of this method and error it is returning is confusing🤔 it looks like it returns the task number and not the day